packages/vinxi/lib/build.js

```diff
@@ -34,7 +33,7 @@ const require = createRequire(import.meta.url);
  */
 export async function createBuild(app, buildConfig) {
 	const { existsSync, promises: fsPromises, readFileSync } = await import("fs");
-	const { join } = await import("path");
+	const { join } = await import("pathe");
```

Comment:
These imports would fail because character would get escaped generating paths like `C:goodjobwindows`
https://github.com/nksaraf/vinxi/blob/505dd51fd130dc6747de2ff5259ef0cf86f62b6f/packages/vinxi/lib/build.js#L457-L460